tfhe/src/high_level_api/keys/inner.rs

```diff
@@ -447,3 +450,91 @@ impl IntegerCompressedCompactPublicKey {
         }
     }
 }
+
+pub struct IntegerServerKeyConformanceParams {
+    pub sk: PBSParameters,
+    pub cpk_param: Option<KeySwitchingKeyConformanceParams2>,
+    pub compression_param: Option<CompressionParameters2>,
```

Comment:
naming, also do we need conformance params or could we just take the parameters used to build the keys and then convert as necessary ? would be much easier for the BC use case in that case I believe ?

Reply:
There is a `impl From<(PBSParameters, CompressionParameters)> for CompressionConformanceParameters` (`CompressionParameters2` renamed to `CompressionConformanceParameters`) for that
But this conversion could indeed be hidden

Reply:
easier to plug from parameters IMO, avoids confusion for hte user potentially

Reply:
Hidden